morphic.js

```diff
@@ -5124,6 +5159,11 @@ CursorMorph.prototype.processKeyDown = function (event) {
     this.target.escalateEvent('reactToKeystroke', event);
 };
 
+CursorMorph.prototype.processKeyUp = function (event) {
+    this.world().currentKey = null;
```

Comment:
My bad! But actually the one needs to stay is the one in `ClipboardHandler`, we can remove this one.